LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
@easternbloc I'll change it to v2 now but I copied this from the hof-boilerplate so will need changing there too. 
I think that has  been mentioned before too but to be honest I have no clue how licensing works or how you do it!